### PR TITLE
fix: add defaults, fix broken script and correct doc

### PR DESCRIPTION
**Pull request by Chr1stian**

### CODE REVIEW

File: .github/README.md [hunk 1]
```diff
@@ -35,9 +38,11 @@ on:
...
 jobs:
   update-version:
-    uses: entur/gha-artifactory/.github/workflows/update-version.yml@v1
+    permissions:
+    contents: write
+    uses: entur/gha-artifactory/.github/actions/update-version.yml@v1
 
   publish-artifact:
-    uses: entur/gha-artifactory/.github/workflows/publish-artifact.yml@v1
+    uses: entur/gha-artifactory/.github/actions/publish-artifact.yml@v1
```
Comment (bvestli) [inline .github/README.md:46]: Supposed to be non existent action? And the correct `uses` should be `entur/gha-artifactory/.github/actions/publish-artifact@v1` ?

File: .github/README.md [hunk 2]
```diff
@@ -35,9 +38,11 @@ on:
 
 jobs:
   update-version:
-    uses: entur/gha-artifactory/.github/workflows/update-version.yml@v1
+    permissions:
+    contents: write
+    uses: entur/gha-artifactory/.github/actions/update-version.yml@v1
```
Comment (bvestli) [inline .github/README.md:43]:
```suggestion
    uses: entur/gha-artifactory/.github/actions/update-version@v1
```

File: .github/README.md [hunk 3]
```diff
@@ -35,9 +38,11 @@ on:
 
 jobs:
   update-version:
-    uses: entur/gha-artifactory/.github/workflows/update-version.yml@v1
+    permissions:
+    contents: write
```
Comment (AlexanderBrevig) [inline .github/README.md:42]:
```suggestion
    permissions:
      contents: write
```

File: .github/actions/update-version/action.yml [hunk 4]
```diff
@@ -44,7 +46,7 @@
         run: |
```
Comment (AlexanderBrevig) [inline .github/actions/update-version/action.yml:46]: This should be modernized to
```bash
major=$(echo "$RELEASE_VERSION" | cut -d. -f1)
minor=$(echo "$RELEASE_VERSION" | cut -d. -f2)
patch=$(echo "$RELEASE_VERSION" | cut -d. -f3)
patch=$(("$patch" + 1))
version="$major.$minor.$patch"
echo "VERSION=$version" >> $GITHUB_ENV
git config user.name github-actions[bot]
git config user.email 41898282+github-actions[bot]@users.noreply.github.com
git add "${GHA_ARTIFACTORY_VERSION_FILE_NAME}"
git commit -m "chore(bump): Bump ${GHA_ARTIFACTORY_VERSION_VARIABLE_NAME} to ${GHA_ARTIFACTORY_VERSION} [skip ci]"
git push origin "$GHA_ARTIFACTORY_GITHUB_REF_NAME" --force
```

https://www.shellcheck.net/ approves of this message

**Review by AlexanderBrevig — CHANGES_REQUESTED** [review]
Looking good!

**Review by AlexanderBrevig — APPROVED** [review]

**Pull request merged**